dali/kernels/imgproc/warp/affine.h

```diff
@@ -32,6 +32,11 @@ struct AffineMapping {
   inline vec<dim> operator()(const vec<dim> &v) const {
     return affine(transform, v);
   }
+
+  DALI_HOST_DEV
+  AffineMapping invert() const {
+    return AffineMapping<dim>(affine_mat_inv(transform));;
+  }
```

Comment:
The name _invert_ suggests an in-place operation. Here, the function returns _inverse_ transform.

Reply:
done